meta-ostro/conf/distro/include/ostro-os-development.inc

```diff
@@ -1,8 +1,10 @@
 # Use the pre-generated keys for IMA signing.
 IMA_EVM_KEY_DIR = "${IMA_EVM_BASE}/data/debug-keys"
 
-# Enable local root access.
-OSTRO_EXTRA_IMAGE_FEATURES += "autologin"
+# Enable local root access. Always use _append, to
+# avoid getting this change overwritten by a
+# OSTRO_IMAGE_EXTRA_FEATURES = "foo" in local.conf.
+OSTRO_IMAGE_EXTRA_FEATURES_append = " autologin"
```

Comment:
as before, we can keep going like this to get it merged. But future should have images where login are confined to a container.